charts/v3-oracle/README.md

```diff
@@ -1,13 +1,13 @@
 # Oracle Helm Chart
 
 Kubernetes Helm chart for deploying oracle - the entity which is responsible for reading StakeWise pool validators data from beacon chain and submitting
-it to the [Oracles](https://github.com/stakewise/contracts/blob/master/contracts/Oracles.sol) smart contract.
+it to the Oracles smart contract.
```

Comment:
Kubernetes Helm chart for deploying [V3 oracle](https://github.com/stakewise/v3-oracle)